local-volume/helm/provisioner/templates/provisioner.yaml

```diff
@@ -0,0 +1,75 @@
+{{- $configMapName := .Values.configmap.configMapName }}
+{{- $provisionerNamespace := .Values.common.namespace }}
+{{- $provisionerName := .Values.daemonset.name }}
+{{- $imageFull := .Values.daemonset.image }}
+{{- $imagePullPolicy := .Values.daemonset.imagePullPolicy }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $configMapName }} 
+  namespace: {{ $provisionerNamespace }} 
+data:
+  storageClassMap: |
+#
+# Populating entries with user provided disk discovery directories 
+# and mount directories per storage class.
+#
+{{- range $tmp, $val := .Values.configmap.storageClass }}
+   {{- range $key, $class := $val }}
+      {{ $key }}:     
+         {{- range $val1, $val2 := $class }}
+           {{ $val1 }}: {{ if $val2 }} {{ $val2 }} {{ else }} {{ printf "/mnt/%s" $key }}  {{ end }}
```

Comment:
Maybe it would be simpler if no mount path is specified, then just use the hostdir

Reply:
done